scout/parse/variant/variant.py

```diff
@@ -282,8 +283,9 @@ def parse_variant(
     cosmic_ids = set()
     for parsed_transcript in parse_transcripts(raw_transcripts):
         parsed_transcripts.append(parsed_transcript)
-        for dbsnp in parsed_transcript.get("dbsnp", []):
-            dbsnp_ids.add(dbsnp)
+        for dbsnpkey in ["dbsnp", "rs_dbSNP150", "rs_dbSNP"]:
+            for dbsnp in parsed_transcript.get(dbsnpkey, []):
+                dbsnp_ids.add(dbsnp)
```

Comment:
What about duplicates? What if rs_dbSNP150 is rs12345 and rs_dbSNP is also rs12345?

Reply:
Right, I guess that's why its a set, though it is not super likely that someone would have both?

Reply:
ah yes, of course, it's using add and not append. Sorry, lack of sleep!